src/appleseed/renderer/modeling/camera/camera.h

```diff
@@ -170,10 +170,11 @@ class APPLESEED_DLLSYMBOL Camera
     // Return a camera representation suitable for rasterization.
     virtual RasterizationCamera get_rasterization_camera() const = 0;
 
-  protected:
+  private:
     struct Impl;
     Impl* impl;
 
+  protected:
```

Comment:
Move protected section ahead of private one so there's a logical progression (public -> protected -> private).

Reply:
I prefer the private implementation to be the first member of the class. I think it would be surprising if it wasn't.
Maybe we can make an exception in cases like this?
Alternatively, I can make it protected again. I like it a bit less, but I still prefer impl to be first.